src/index.js

```diff
@@ -0,0 +1,186 @@
+const addTasksBtn = document.getElementById('add-task-btn');
+const deskTaskInput = document.getElementById('description-task');
+const todoWrapper = document.querySelector('.todos-wrapper');
+
+let tasks;
+/* eslint-disable */
+!localStorage.tasks ? tasks = [] : tasks = JSON.parse(localStorage.getItem('tasks')); 
```

Comment:
I really love the way you conditionally get data from the localStorage here. I love the syntax:+1:

Reply:
Thanks!